openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -51,6 +51,12 @@ def find_paths_by_hash(texture_hash):
         str: path to texture if found.
 
     """
+    if AYON_SERVER_ENABLED:
+        raise ValueError(
```

Comment:
What about changing this to `KnownPublishError`? Because we sort of know that it might happen here :)

```suggestion
        raise KnownPublishError(
```